cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -612,6 +688,8 @@ def test(
     requests=Resources(mem="1Gi"),
     limits=Resources(mem="1Gi"),
 )
+
+# Now, we define a function that consolidates the ``data_preparation``, ``train`` and ``test`` methods, and returns the sales predictions.
```

Comment:
We shouldn't add content between decorator and function as I have specified previously. Let's remove this.